hack/errata-check.sh

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# RED HAT INTERNAL USE ONLY
+#
+# Purpose
+# -------
+# Which PRs are in the build currently attached to the errata for the next
+# release?
+#
+# The script will query Errata Tool for the active errata based on the
+# branch specified.  It will then look up the build attached and
+# determine the WMCO commit the build was built from.  Finally, it will list the
+# merge commits since the last WMCO release for that version.  The commit
+# associated with the build will be tagged with a shortened build NVR,
+# 'wmco-container-X.Y.Z-n'.
+#
+# Install Required Tools
+# ----------------------
+# dnf install curl jq gh
+#
+# Usage
+# -----
+# * VPN connectivity and valid kerberos ticket required
+# * Update your local clone and branches
+# * Run this script ./errata-check.sh <branch_name>
```

Comment:
nit: assume that this is ran from repo root
`./hack/errata-check.sh <branch_name>`

Reply:
Updated.